app/assets/javascripts/discourse/app/widgets/do-not-disturb.js

```diff
@@ -11,21 +11,10 @@ export default createWidget("do-not-disturb", {
   saving: false,
 
   html() {
-    if (this.currentUser.isInDoNotDisturb()) {
-      return [
-        h("button.btn-flat.do-not-disturb-inner-container", [
-          iconNode("toggle-on"),
-          this.label(),
-        ]),
-      ];
-    } else {
-      return [
-        h("button.btn-flat.do-not-disturb-inner-container", [
-          iconNode("toggle-off"),
-          h("span.do-not-disturb-label", I18n.t("pause_notifications.label")),
```

Comment:
I do not see this string used anymore if the "do not disturb" mode is not enabled. Can you confirm it is still in use?

Reply:
We use it in a couple of places: below in the `_label` function and also in the `profile-tab-content` component.